tools/commands.py

```diff
@@ -46,23 +52,66 @@ def data_collector(config, catch=True):
 #             raise e
 
 
-def package_manager(config, commands, catch=False, force=False, default_git_branch=TENTACLES_DEFAULT_BRANCH):
+def _check_arg_and_cleanup(command_args, argument):
+    if argument in command_args:
+        command_args.pop(command_args.index(argument))
+        return True
+    return False
+
+
+async def _handle_package_manager_command(command_args, aiohttp_session=None):
```

Comment:
I think this should be defined in a tentacle manager entrypoint

Reply:
I was thinking about it and ended up adding it in OctoBot because it's using OctoBot's command line, like this tentacles-manager is just a lib and the entrypoint is only it's caller. You think we should share this command line management with the tentacles-manager ?

Reply:
it seemed to me that we had talked about the possibility of using the Tentacle Manager standalone. So with a CLI.